vulcan/lib/client/jsx/components/workflow/session/__tests__/session_manager.spec.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import {VulcanProvider} from '../../../../contexts/vulcan_context';
+import {rest} from 'msw';
+import {setupServer} from 'msw/node';
+import {render, fireEvent, waitFor, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import SessionManager from '../session_manager';
+import {stateFromActions} from '../../../../test_utils/state';
+import {
+  createStatusFixture,
+  createStepFixture,
+  createStepStatusFixture,
+  createWorkflowFixture
+} from '../../../../test_utils/fixtures';
+import {
+  setDownloadedData,
+  setStatus,
+  setWorkflow,
+  setWorkflows,
+  addValidationErrors
+} from '../../../../actions/vulcan';
+
+describe('SessionManager', () => {
+  const workflow = createWorkflowFixture({
```

Comment:
Cool!  I intend to actually improve these fixture functions and give them a nicer "builder" like interface with some sane defaults and the ability to do things like autocompute the `dependencies_of_outputs` inner piece.